app/src/models/diff.ts

```diff
@@ -118,18 +120,27 @@ export interface IBinaryDiff {
   readonly kind: DiffType.Binary
 }
 
-export interface IDiffTooLarge {
-  readonly kind: DiffType.TooLarge
-  /**
-   * The length of the diff output from Git which exceeds the runtime limits:
-   *
-   * 268435441 bytes = 256MB - 15 bytes
-   */
-  readonly length: number
+export interface ILargeTextDiff {
+  readonly kind: DiffType.LargeText
+  /** The unified text diff - including headers and context */
+  readonly text?: string
```

Comment:
does the `text` value need to be optional? looks like the only time we call it, it is set

Reply:
Oh you're right! That's a holdover to the old way of doing things .